### PR TITLE
Connector changes to support composite types

### PR DESCRIPTION
**Pull request by vaibhav-yb**
This PR adds the connector support to process the composite data types as string and publish the records to Kafka topics. A basic unit test has also been added to verify the change.

This PR closes issue #92

### CODE REVIEW

File: src/test/java/io/debezium/connector/yugabytedb/YugabyteDBUserDefinedTypesTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  private final String INSERT_FORMAT = "INSERT INTO udt_table VALUES (%s, ('Vaibhav', 'Kushwaha'));";
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    // Make sure this test is using the YugabyteDB version which has the changes to support
+    // composite types with Change data capture
+    // ybContainer = TestHelper.getYbContainer();
+    // ybContainer.start();
+
+    // TestHelper.setContainerHostPort(ybContainer.getHost(), ybContainer.getMappedPort(5433));
```
Comment (vaibhav-yb) [inline src/test/java/io/debezium/connector/yugabytedb/YugabyteDBUserDefinedTypesTest.java:44]: Commented this to use a local build with the server side changes for UDT support. Will uncomment and change the docker image version so as to use the latest image with all the necessary changes.

File: src/test/resources/yugabyte_user_defined_types.ddl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CREATE TYPE my_name_type AS (first_name text, last_name varchar(40));
+
+CREATE TABLE udt_table (id INT PRIMARY KEY, name_col my_name_type);
```
Comment (arpang) [inline src/test/resources/yugabyte_user_defined_types.ddl:3]: missing new line at the end of file

Comment (arpang) [inline src/test/resources/yugabyte_user_defined_types.ddl:3]: replace udt with composite_type

**Review by arpang — CHANGES_REQUESTED** [review]
A general comment throughout the diff:

UDT refers to a type defined by the user and can be of any kind (composite, enum, range, domain etc). This diff specifically adds support for composite types. So let's replace UDT with Composite type for clarity throughout the diff.

File: src/test/resources/yugabyte_user_defined_types.ddl (hunk 2 above)
Comment (vaibhav-yb) [reply src/test/resources/yugabyte_user_defined_types.ddl]: Done. I have removed the complete file actually, not dependent on a ddl file now.

File: src/main/java/io/debezium/connector/yugabytedb/PgOid.java [hunk 3]
```diff
@@ -41,4 +41,5 @@ public final class PgOid extends Oid {
     public static final int INT8RANGE_OID = 3926;
```
[the remark this answers is not in the document]
Comment (vaibhav-yb) [reply src/main/java/io/debezium/connector/yugabytedb/PgOid.java:41]: Done.

File: src/main/java/io/debezium/connector/yugabytedb/PgOid.java [hunk 4]
```diff
@@ -41,4 +41,5 @@ public final class PgOid extends Oid {
...
     public static final int INT8RANGE_ARRAY = 3927;
     public static final int ENUM_OID = 3500;
+    public static final int UDT_OID = 2249;
```
[the remark this answers is not in the document]
Comment (vaibhav-yb) [reply src/main/java/io/debezium/connector/yugabytedb/PgOid.java]: Renamed.

**Review by vrajat — APPROVED** [review]

**Pull request merged**